corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -140,7 +140,20 @@
                 $('form[name="exportFiltersForm"]'));
             $scope.prepareExportError = null;
             $scope.preparingExport = true;
-            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {"Export type": $scope.exportList[0].export_type});
+            var userTypes = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            function getFilterName(exportType) {
+                return (exportType === "form" ? "emw" : "case_list_filter");
+            }
+            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {
+                "Export type": $scope.exportList[0].export_type,
+                "filters": _.map(
+                    $("#exportFiltersFormId input[name=" + getFilterName($scope.exportList[0].export_type) + "]")
+                        .serializeArray()[0].value.split(','),
+                    function (item) {
+                        if (item[0] === "t") { return userTypes[item.substring(3)]; }
```

Comment:
theoretically it seems that other fields could also start with a `t`. maybe check for `t__` or limit it to the one field you know has fields formatted like this?

Reply:
Good point! I will also change this in the other pr:

https://github.com/dimagi/commcare-hq/pull/22084/commits/ef0df409227babfd0ec2265f01fc70c8a0e9f58c